.github/workflows/ci.yaml

```diff
@@ -66,8 +66,12 @@ jobs:
           git diff --name-only --exit-code go.sum || ( echo "Run go tidy" && false )
 
   packet:
-    name: packet
+    name: packet (Calico ${{ matrix.calico }})
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        calico: ["off", "on"]
```

Comment:
Might it not make more sense semantically to have this matrix be a matrix for CNI plugin, where calico-vpp is just one of the choices?  That would let us have an easy visible path for other CNIs to be added later.  Thoughts?